packages/default/scss/toolbar/_theme.scss

```diff
@@ -1,3 +1,14 @@
+$toolbar-separator-border: inherit !default;
+
+$toolbar-button-expanded-bg: null !default;
+$toolbar-button-expanded-shadow: none !default;
+$toolbar-button-hovered-opacity: .05 !default;
+$toolbar-button-active-opacity: .22 !default;
+$toolbar-button-focused-opacity: 0 !default;
+$toolbar-button-focused-outline-opacity: .08 !default;
+
+$toolbar-button-mode: flat !default;
```

Comment:
This is probably a good idea, but it would be great if the values for this variable are documented. Looking at the code below these are `flat` and `thin`, and it seems that `thin` means that the default state is invisible, but it would be great if a comment explained this.